.github/workflows/tests.yml

```diff
@@ -0,0 +1,352 @@
+name: 'Tests'
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  lint:
+    name: 'lint (node: ${{ matrix.node }})'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install modules
+        run: yarn
+      - name: Run lint
+        run: yarn run -s lint
+
+  unit_back:
+    name: 'unit_back (node: ${{ matrix.node }})'
+    needs: [lint]
+    runs-on: ubuntu-latest
+    env:
+      CODECOV_TOKEN: ${{ secrets.codecov }}
+    strategy:
+      matrix:
+        node: [12, 14]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install modules
+        run: |
+          yarn && yarn global add codecov
+          echo "$(yarn global bin)" >> $GITHUB_PATH
+      - name: Run test
```

Comment:
"Run test**s**" ? (consistency with other places in the file. Also there is many test suites running here)
![image](https://user-images.githubusercontent.com/25851739/95996090-9243c280-0e32-11eb-9340-e2dd0f12752e.png)